app/src/main/java/com/mapbox/dash/example/MainActivity.kt

```diff
@@ -444,6 +463,18 @@ class MainActivity : DrawerActivity() {
             controller.stopNavigation()
         }
 
+        bindSwitch(
+            switch = menuBinding.customCompassDataInputs,
+            state = setCustomCompassDataInputs,
+        ) { enabled ->
+            sampleSensorEventManager?.compassData?.observeWhenStarted(this) {
+                if (enabled) {
+                    Log.d(TAG, "Updating compass data: $it")
+                    Dash.controller.updateCompassData(it)
```

Comment:
Is there a way to reset the data when the switch is unchecked?

Reply:
No